ProjectSource/app-library/src/test/java/com/kayako/sdk/android/k5/kre/base/KreConnectionFactoryTest.java

```diff
@@ -0,0 +1,24 @@
+package com.kayako.sdk.android.k5.kre.base;
+
+import static org.hamcrest.CoreMatchers.notNullValue;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ErrorCollector;
+
+public class KreConnectionFactoryTest {
+
+    @Rule
+    public final ErrorCollector errorCollector = new ErrorCollector();
+
+    @Test
+    public void getConnection() {
+        final boolean agentTrue = Boolean.TRUE;
+        final boolean agentFalse = Boolean.FALSE;
+        final KreConnection sAgentKreConnection =
+                KreConnectionFactory.getConnection(agentTrue);
+        final KreConnection sMessengerKreConnection =
+                KreConnectionFactory.getConnection(agentFalse);
+        errorCollector.checkThat(sAgentKreConnection, notNullValue());
+        errorCollector.checkThat(sMessengerKreConnection, notNullValue());
```

Comment:
⚠️  Consider using AAA("Arrange-Act-Assert") pattern for arranging and formatting code in JUnit test methods.  

**[More details](https://confluence.devfactory.com/x/MRikEw)**
          
## The possible solution in this case

```Java  

// Arrange
final boolean agentTrue = Boolean.TRUE; 
final boolean agentFalse = Boolean.FALSE; 

// Act
final KreConnection sAgentKreConnection = 
		KreConnectionFactory.getConnection(agentTrue); 
final KreConnection sMessengerKreConnection = 
		KreConnectionFactory.getConnection(agentFalse); 

// Assert
errorCollector.checkThat(sAgentKreConnection, notNullValue()); 
errorCollector.checkThat(sMessengerKreConnection, notNullValue());
```